### PR TITLE
Fix integrator customise documentation

### PR DESCRIPTION
**Pull request by gberaudo**

### CODE REVIEW

**Comment by gberaudo** [comment]
Travis failure seams unrelated to this PR.
Added a pointer to CONST_vars.yaml.

File: doc/integrator/customise.rst [hunk 1]
```diff
@@ -4,34 +4,39 @@
...
 * ``date``
 * ``time``
 * ``datetime`` `see the python-dateutil documentation <http://labix.org/python-dateutil#head-b95ce2094d189a89f80f5ae52a05b4ab7b41af47>`_
-* ``url`` see bellow
+* ``url`` see below
+
+Check ``CONST_vars.yaml`` for examples of usage.
+
+In order to inherit the default values from ``CONST_vars.yaml`` make sure the ``update_paths`` section contains
+the item ``admin_interface.available_functionnalities`` / respectively ``admin_interface.available_metadata``.
```
Comment (sbrunner) [inline doc/integrator/customise.rst]: `/` => `or`?

Comment (gberaudo) [inline doc/integrator/customise.rst]: OK, done

**Comment by sbrunner** [comment]
Looks good, merge when it's OK :-)

**Comment by gberaudo** [comment]
Thank you for the review.

**Pull request merged**